.github/workflows/vib-verify.yaml

```
@@ -0,0 +1,81 @@
+name: 'Early feedback'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+      - bitnami:master
+    types:
+      - assigned
+      - opened
+      - synchronize
+      - reopened
```

Remove "assigned", it doesn't make sense to run the action after assigning a PR:
```suggestion
    types:
      - opened
      - synchronize
      - reopened
```

---

If you apply this ⬆️  , then you can just skip the `types`, as these are the defaults for [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)